gestalt-inject-java/src/main/java/org/terasology/gestalt/annotation/processing/BeanDefinitionProcessor.java

```diff
@@ -66,13 +69,23 @@ public class BeanDefinitionProcessor extends AbstractProcessor {
     };
     private final Set<String> processed = new HashSet<>();
     private final Set<String> beanDefinitions = new HashSet<>();
+    /**
+     * the elements.
+     */
     protected Elements elementUtils;
+    /**
+     * the types.
+     */
     protected Types typeUtils;
     private Filer filer;
     private ServiceTypeWriter writer;
     private ElementUtility utility;
     private Messager messager;
 
+    /**
+     * Init proocessingEnvironment
+     * @param processingEnvironment the ProcessingEnvironment.
```

Comment:
```suggestion
     * {@inheritDoc}
```

This is an override from an already well-documented base method, so we might as well inherit that.